doc/g191.md

```diff
@@ -0,0 +1,55 @@
+
+## CONTENTS
+
+1. [General](#general)
+2. [Software tools](#software_tools)
+3. [License and copyright](#license_and_copyright)
+
+[Annex A](./g191_Annex_A.md) – List of software tools available
+
+[Annex B](./g191_Annex_B.md) – ITU-T software tools General Public License
+
+
+Electronic attachment: STL2009 Software Tool Library and manual
+
+# Recommendation ITU-T G.191 - Software tools for speech and audio coding standardization
+
+## 1. General<a name="general"></a>
+
+In the process of generating speech and audio coding standards, the following situations often happen:
+
+* In many cases, experimental results generated with different software tools may not be directly compared.
+
+* Software tools used by different organizations may not perfectly conform to related ITU‑T Recommendations, which may delay ITU-T standardization processes.
+
+* ITU-T Recommendations may leave scope for different implementations.
+
+* New speech and audio coding standards are increasing in complexity, leading to non‑bitexact specifications; furthermore, appropriate testing procedures to assure interoperability of different implementations are needed.
+
+The need for a common set of tools has been recognized in past ITU-T standardization activities of speech algorithms. As a consequence, a library of portable, interworkable and reliable software routines has been established.
+
+## 2. Software tools<a name="software_tools"></a>
+
+To clarify the use of the set of software tools, arranged as a software tool library, the ITU-T makes the following recommendations:
+
+1) The software tools specified in Annex A should be used as building modules of signal processing blocks to be used in the process of generation of ITU-T Recommendations, particularly those concerned with speech and audio coding algorithms.
+
+2) Some of the tools shall be used in procedures for the verification of interoperability of ITU‑T standards, mainly of speech and audio coding algorithms whose description is in terms of non-bitexact specifications.
+
+3) The use of these modules should be made strictly in accordance with the technical instructions of their attached documentation, and should respect the following terms.
+
+## 3. License and copyright<a name="license_and_copyright"></a>
```

Comment:
Is the hyperlink necessary?

Reply:
@dennisguse. Not really. I inserted them at the beginning as I had one big document. 
Now that it is split into three I can remove the hyperlinks if you think it is a bad idea.